book/src/pong_tutorial/pong_tutorial_02.md

```diff
@@ -0,0 +1,367 @@
+# Drawing the paddles
+
+Now let's do some drawing! But to draw something, we need something to draw. In 
+Amethyst, those "somethings" are called Entities, which are described by 
+Components.
+
+Amethyst uses Specs for its ECS (Entity-component system), which is a parallel 
+Entity-component system written in Rust. You can learn more about Specs in the 
+[The Specs Book][sb].
+
+## A quick refactor
+
+Let's create a new file called `pong.rs` to hold our core game logic. We can 
+move the `Pong` struct over here, and the `impl State for Pong` block as well. 
+Then, in `main.rs` declare a module:
+
+```rust,ignore
+mod pong; 
+```
+
+And in the `run()` function add:
+
+```rust,ignore
+use pong::Pong;
+```
+
+Now you can just delete various `main.rs` use statements until the Rust compiler 
+stops complaining about unused imports. In `pong.rs` we'll need these use 
+statements to make it through this chapter:
+
+```rust,ignore
+use amethyst::prelude::*;
+use amethyst::ecs::{Component, DenseVecStorage};
+use amethyst::assets::Loader;
+use amethyst::core::cgmath::Vector3;
+use amethyst::core::transform::{LocalTransform, Transform};
+use amethyst::renderer::{Camera, Material, MaterialDefaults, PosTex, MeshHandle, 
+                         Event,KeyboardInput, VirtualKeyCode, WindowEvent};
+```
+
+## Our first Component
+
+In pong.rs let's create our first `Component`, a definition of a paddle.
```

Comment:
An uninitiated reader might go "wait what's a component?" here, so maybe a quick blurb about what the term `Component` means would be helpful.

Reply:
I think instead we can make it more obvious above that the Specs Book is mandatory reading maybe?

Reply:
That works too, linking to it would help in that regard as well.